### DISCUSSION TITLE
Github Graduation not PR'ing?

### DISCUSSION THREAD

**Discussion by kedarabhyankar** [community/community#17773] [GitHub Education]
Kind of confused, I pushed the correct commit and updated all the files as necessary, removing the directories as needed, but the PR is saying that it's looking for the original user's folder and not mine. I don't see any traces of the original user's files in my directory.

**Comment by elisemoe** [comment]
Congrats on your graduation @kedarabhyankar! 💖 

When I look at the changed files, I [see you've deleted another participant's folder and markdown file](https://github.com/education/GitHubGraduation-2022/pull/1682/files) in addition to adding yours. You should not need to remove any directory. 

Please submit another Pull Request only adding your files.

![Screen Shot 2022-05-03 at 8 26 16 AM](https://user-images.githubusercontent.com/6633808/166484567-417bd286-3f83-4211-9805-e51f7c168ca2.png)
